DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -124,6 +124,7 @@
 #include "DataFormats/SiStripDetId/interface/StripSubdetector.h"
 #include "DataFormats/MuonDetId/interface/MuonSubdetId.h"
 #include "DataFormats/ForwardDetId/interface/MTDDetId.h"
+#include "DataFormats/Scouting/interface/Run3ScoutingHitPatternPOD.h"
```

Comment:
(Loosely related to [Matti's comment](https://github.com/cms-sw/cmssw/pull/37149/files#r820737323))

Seems to me that `<use name="DataFormats/Scouting"/>` would have to be included now in `DataFormats/TrackReco/BuildFile.xml`.